src/components/Home.js

```diff
@@ -184,6 +185,35 @@ function Home() {
             Click here to join
           </Link>
         </div>
```

Comment:
Can we move the new block above the 'How to Join' so it comes first as people scroll